src/widgets/widget.ts

```diff
@@ -63,13 +67,37 @@ export class Widget {
     }
 
     public async waitToBePresent(timeout: number = 500): Promise<void> {
-        return this.elem.waitToBePresent(timeout);
+        return this.elem.waitUntil(async () => await this.isPresent(), timeout);
     }
 
     public async waitToBeNotPresent(timeout: number = 500): Promise<void> {
         return this.elem.waitUntil(async () => !(await this.isPresent()), timeout);
     }
 
+    public async waitToBeDisplayed(timeout: number = 500): Promise<void> {
```

Comment:
Could we use a "global const" for the default timeout value of all methods -as there starts to be quite a lot-, or that would not have an added value?

Reply:
Absolutely, this makes a lot of sense.
Updated code to add a "DefaultTimeout" class with 2 constants for the different types of timeouts for waits: fast (set to 500ms) and slow (set to 1000ms). Updated also widget and page object wait methods to use this new constants.